script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -96,14 +96,14 @@ To be honest, I couldn't stand that smile anymore.
 "Why do you have lines all over your body?
 And there are so many cracks in this room as well, it looks like it's about to fall apart."
 <Page27>
-Although his smile vanished for a brief moment, he quickly returned to his smiling face and stood up from his chair.
+Although his smile vanished for a brief moment, he quickly returned to his smiling face and stood up from his chair.//'he quickly returned to his smiling face' sounds weird, needing rephrasing?
```

Comment:
Although his smile vanished for a brief moment, it quickly returned and he stood up from his chair.